README.md

```diff
@@ -0,0 +1,41 @@
+# The .NET Foundation Membership Committee
+
+The committee is responsible for evaluating new member applications and discussing membership criteria.
+
+## Details
+
+**Chairperson(s):** [Bill Wagner](https://github.com/BillWagner)
+
+**Meets:** 1st and 3rd Tuesday of each month at 1:30pm UTC
+
+**Repository:** https://github.com/dotnet-foundation/wg-membership
+
+## Governance
+
+The meeting notes are available [here](meetings).
+
+This working group seeks to have a diverse set of representatives from various stakeholders. At the same time, we try to keep the working group somewhat small so that we can have productive meetings. But in order to satisfy the desire for open participation from everyone – and in the sake of transparency – we made this repo public, which includes notes and proposals that documents the progress of the working group. If you have any questions, please open a discussion.
+
+## Contributing
+
+### Raising Ideas
+
+If you have an idea on how the membership offering could be enhanced or any issues you think should be addressed with the current membership offering, please raise an issue on this GitHub Repository
+
+### Attending Membership Meetings
+
+_Info on how to join the committee here._
```

Comment:
Good point @wordshaker 

I think the easiest way to do this is to raise an issue asking to join. I'll watch for those, and add people to meeting invites.

Once this is added, this is ready to merge